src/sensor.cpp

```diff
@@ -766,8 +766,9 @@ namespace librealsense
         {
             return static_cast<rs2_stream>(RS2_STREAM_GPIO);
         }
-
+#ifndef __APPLE__
```

Comment:
Please explain in a comment

Reply:
An error in the log about an incorrect gpio appears, but this does not affect the behavior of the code in any way. Perhaps this is due to the fact that we receive data bypassing the usual scheme. I think this can be left as a temporary way.